src/main/resources/db/migration/V1_48__update_judicial_service_code_mapping.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO judicial_service_code_mapping (service_id,ticket_code,service_code,service_description)
+VALUES (117,'374','BCA1','Care Standards'),
+       (118,'374','BCA2','Mental Health'),
+       (119,'374','BCA3','Primary Health Lists'),
+       (120,'374','BCA4','Special Educational Needs'),
+       (121,'376','BBA2','Criminal Injuries Compensation');
+
+COMMIT;
```

Comment:
Request @manukundloo-hmcts can we please verify the same scripts should be updated in integration test cases as well please :)

Reply:
Done the changes for Functional test cases and also wrote all DB change code in one file